### PR TITLE
Update the readme

### PR DESCRIPTION
**Pull request by alberto512**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,17 @@
 
 A module for Terasology.
 
-## Planed content
+## How to play?
 
-A character called gooey will appear. He offers you existing and dangerous quests. However be warned: He might modify your world not in a way you like it!
+First you have to start the game with GooeysQuests activated and in a Perlin world. When the game is created, you will see a character named Gooey that will ask you something. Depending on your answer, Gooey will change the world adding some structures. Now there are two available structures, the dungeon and the dwarf hall, but they have diferents combinations, so every time gooey add the structure, you will see something new. But be careful, maybe you don’t like these changes.
+
+##How to add more structures?
```
Comment (anuar2k) [inline README.md]: You've missed a space between "##" and "How"

Comment (alberto512) [reply README.md]: oh, ok ,thank you

File: README.md [hunk 2]
```diff
@@ -2,10 +2,17 @@
 
 A module for Terasology.
 
-## Planed content
+## How to play?
 
-A character called gooey will appear. He offers you existing and dangerous quests. However be warned: He might modify your world not in a way you like it!
+First you have to start the game with GooeysQuests activated and in a Perlin world. When the game is created, you will see a character named Gooey that will ask you something. Depending on your answer, Gooey will change the world adding some structures. Now there are two available structures, the dungeon and the dwarf hall, but they have diferents combinations, so every time gooey add the structure, you will see something new. But be careful, maybe you don’t like these changes.
```
[the remark this answers is not in the document]
Comment (vampcat) [reply README.md]: Here is one possible way to phrase this. Keep in mind you can do it differently if you so desire.
```
- Start a game with GooeysQuests activated and in a Perlin world.
- Walk around till you find a character named Gooey (a green cube), and talk to him.
.... and so on.
```

File: README.md [hunk 3]
```diff
@@ -2,10 +2,17 @@
...
+
+## How to add more structures?
```
Comment (vampcat) [inline README.md]: Change this to `Adding more structures`

File: README.md [hunk 4]
```diff
@@ -2,10 +2,17 @@
...
+
+If you want to add a new structure, first you have to create the structures templates that will form the structure. Then you will have to create at least two java class GooeysQuests /src/ main/ java/ org/ terasology/ gooeysQuests/ quests. One will be the component and the other will be the system that allow Gooey to spawn your structure.
```
Comment (vampcat) [inline README.md]: The paths look very weird. Change it to this format:
\`/src/main/java/org/terasology/...\` . That will make it look like this:
`/src/main/java./...`

Comment (vampcat) [reply README.md]: Further, rephrase the sentence so that the rerader knows *what* he's creating, then he knows *where*. What I mean is this:
"Then you will have to create two java classes: a component and a system, typically in the folder `/src/main/java/...`".

File: README.md [hunk 5]
```diff
@@ -2,10 +2,17 @@
...
+Name conventions:
```
Comment (vampcat) [inline README.md]: Rename this to Naming conventions

File: README.md [hunk 6]
```diff
@@ -2,10 +2,17 @@
...
+•	Add the name of your structure at the beggining of the name of each structure template that you create
```
Comment (vampcat) [inline README.md]: Typo in the word beginning.

File: README.md [hunk 7]
```diff
@@ -2,10 +2,17 @@
...
+•	The format for java class names is: (Yourstructure)QuestComponent and (Yourstructure)QuestSystem
```
Comment (vampcat) [inline README.md]: Change this to \`<YourStructure>QuestComponent\` and \`<YourStructure>QuestSystem\`, so that it is displayed `like this`.

File: README.md [hunk 8]
```diff
@@ -2,10 +2,17 @@
...
 
 ## Credits
 
-A drawing of a "gooey" made by SuperSnark was the inspiration for gooey main character of this module.
+A drawing of a "gooey" made by SuperSnark was the inspiration for Gooey main character of this module.
```
Comment (vampcat) [inline README.md]: `..was the inspiration for Gooey, the main character..`

File: README.md [hunk 9]
```diff
@@ -2,10 +2,17 @@
 
 A module for Terasology.
 
-## Planed content
+## How to play?
```
[the remark this answers is not in the document]
Comment (alberto512) [reply README.md]: ok

File: README.md [hunk 10]
```diff
@@ -2,10 +2,21 @@
...
 
-## Planed content
+## Getting Started
 
-A character called gooey will appear. He offers you existing and dangerous quests. However be warned: He might modify your world not in a way you like it!
+- You have to start the game with GooeysQuests activated and in a Perlin world.
+- Walk around until you find a character named Gooey that will ask you something.
+- Depending on your answer, Gooey will change the world adding some structures.
+- Right now there are two available structures, the dungeon and the dwarf hall, but they have different combinations, so every time gooey add the structure, you will see something new.
+- But be careful, maybe you don’t like these changes.
```
Comment (vampcat) [inline README.md]: Rephrase this to "But be careful. you might not like these changes."

File: README.md [hunk 11]
```diff
@@ -2,10 +2,21 @@
...
+
+## Adding more structures
+
+If you want to add a new structure, first you have to create the structures templates that will form the structure. Then you will have to create two java classes: a component and a system, typically in the folder `/src/main/java/org/terasology/gooeysQuests/quests`
+##### Naming conventions:
+- Add the name of your structure at the beginning of the name of each structure template that you create
+- The format for java class names is: <YourStructure>QuestComponent and <YourStructure>QuestSystem
```
Comment (vampcat) [inline README.md]: Add the grave/backtick symbol around `<YourStructure>QuestComponent` and `<YourStructure>QuestSystem`.

File: README.md [hunk 12]
```diff
@@ -2,10 +2,21 @@
 
 A module for Terasology.
```
Comment (vampcat) [inline README.md]: Actually, elaborate on this line a little more. Instead of just `A module for Terasology` change this to `A module for Terasology that causes a Gooey to randomly spawn around the world, enabling you to create Structures`. This makes the line a lot more meaningful.

File: README.md [hunk 13]
```diff
@@ -2,10 +2,21 @@
...
 
-## Planed content
+## Getting Started
 
-A character called gooey will appear. He offers you existing and dangerous quests. However be warned: He might modify your world not in a way you like it!
+- You have to start the game with GooeysQuests activated and in a Perlin world.
+- Walk around until you find a character named Gooey that will ask you something.
+- Depending on your answer, Gooey will change the world adding some structures.
+- Right now there are two available structures, the dungeon and the dwarf hall, but they have different combinations, so every time gooey add the structure, you will see something new.
```
Comment (jellysnake) [inline README.md]: Rephrase to "every time Gooey adds a structure"

File: README.md [hunk 14]
```diff
@@ -2,10 +2,21 @@
 
 A module for Terasology.
 
-## Planed content
+## Getting Started
 
-A character called gooey will appear. He offers you existing and dangerous quests. However be warned: He might modify your world not in a way you like it!
+- You have to start the game with GooeysQuests activated and in a Perlin world.
```
Comment (jellysnake) [inline README.md]: The world doesn't have to be perlin, it just needs to have substantial enough cliffs/mountains/hills to spawn the dwarf halls in.  Pelin is an example of a world generator that has these features.

File: README.md (hunk 11 above)
Comment (jellysnake) [reply README.md]: for reference that is this `` ` ``

**Comment by Cervator** [comment]
Closing as I figure #28 is a replacement for this PR
